orchestrator-salt/src/main/resources/salt/salt/java/scripts/set_default_java_version.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -n "$TARGET_JAVA_VERSION" ]; then
+  CURRENT_JAVA_VERSION=$(java -version 2>&1 | grep -oP "version [^0-9]?(1\.)?\K\d+" || true)
+
+  if [ -z "$CURRENT_JAVA_VERSION" ]; then
+    echo "Failed to determine the current java version."
+    exit 1;
+  elif [ $CURRENT_JAVA_VERSION -eq $TARGET_JAVA_VERSION ]; then
+    echo "Current java version equals to the target version."
+    exit 0;
+  elif [ $CURRENT_JAVA_VERSION -eq 8 ] && [ $TARGET_JAVA_VERSION -eq 11 ]; then
+    echo "Change java version from $CURRENT_JAVA_VERSION to $TARGET_JAVA_VERSION"
+
+    alternatives --set java java-11-openjdk.x86_64
+    ln -sfn /etc/alternatives/java_sdk_11 /usr/lib/jvm/java
+    mkdir -p /etc/alternatives/java_sdk_11/jre/lib/security
+    ln -sfn /etc/alternatives/java_sdk_11/conf/security/java.security /etc/alternatives/java_sdk_11/jre/lib/security/java.security
+    ln -sfn /etc/pki/java/cacerts /etc/alternatives/java_sdk_11/jre/lib/security/cacerts
```

Comment:
is this valid for centos7 and rh8 too?

Reply:
TBH I don't know, this should be covered when start the support of RHEL8 runtime images.

Reply:
Ticket under the RHEL8 runtime image epic to cover: https://jira.cloudera.com/browse/CB-20820